src/components/composite/DeFiPositionCard/fixtures.ts

```
@@ -1,416 +1,455 @@
-  ethStakingRewards: {
-    name: 'ETH Staking Rewards',
-    symbol: 'ETH Staking Rewards',
-    decimals: 18,
-    logo: 'https://tokens.1inch.io/0xabcdef1234567890abcdef1234567890abcdef12.png',
-    address: '0xabcdef1234567890abcdef1234567890abcdef12',
-    chain: ethereumChain,
+  {
+    name: 'Aave V3',
+    assetUsd: 7.04781961968e-9,
+    debtUsd: 1,
+    netUsd: -0.99999999295218038,
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 0.9999999929521803

(lint/correctness/noPrecisionLoss)